src/styles/utils/_normalize.scss

```diff
@@ -0,0 +1,53 @@
+.container {
```

Comment:
Avoid defining styles for generic classes like '.container' in a normalize file. This file is for setting reasonable default styles for all browser-introduced styling. Specific class styling should be done in relevant component files.